src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json

```diff
@@ -5,42 +5,49 @@
     "phone" : "94351253",
     "email" : "alice@example.com",
     "address" : "123, Jurong West Ave 6, #08-111",
+    "remark" : "",
```

Comment:
if remarks are usually empty initially I was thinking can explore having it as an optional parameter, can look into this further

Reply:
I think for something like remarks a safe implementation is to always have the field there, maybe we can choose to fix this on display side (Don't display empty remarks on the GUI). Reason being its probably not worth the hassle to deal with null values or Optionals.